src/prefect/agent/docker/agent.py

```diff
@@ -526,8 +526,7 @@ def populate_env_vars(
         # In the future we don't want the agent to set logging config on a flow
         # run unless explicitly asked.  We do this early on so later config
         # sources can override
-        if run_config is None:
-            env.update({"PREFECT__LOGGING__LEVEL": config.logging.level})
+        env.update({"PREFECT__LOGGING__LEVEL": config.logging.level})
```

Comment:
So my only point of concern is this still
>  # In the future we don't want the agent to set logging config on a flow run unless explicitly asked.  We do this early on so later config sources can override

we are in direct violation of this idea here

`env.update({"PREFECT__LOGGING__LEVEL": config.logging.level})` 

where the logging level from the config.toml on the agent's machine is propagated to the flow run.

Reply:
What's the motivation behind not wanting the agent to propagate its logging level by default?

Reply:
I don't know, this was a decision I presume was made by Jim and we're only maintaining that behavior when run configs are `None` for backwards compatibility. I could see the reasoning that the config.toml may not be expected to affect flow runs for that agent.

Reply:
Hmmm. From my perspective, it seems like the config.toml would provide a reasonable default + doesn't break any existing setups.

I can't imagine myself getting particularly upset about it, besides maybe having to redeploy an agent.

I have the least context here, so I'm open to persuasion.

If we're going to remove this logic, though, we should remove it from all agents and stop supporting it.

Reply:
I also don't quite understand the reasoning. I'm fine with always pulling from the config for the base value, it makes sense to me.